src/components/NcAppSidebar/NcAppSidebar.vue

```diff
@@ -1116,11 +1116,21 @@ export default {
 </script>
 
 <style lang="scss">
+// Allows to use transition over a custom CSS property (CSS Variable)
+// Ignored on old browsers resulting in slightly noticeable jump
+@property --app-sidebar-offset {
```

Comment:
fancy!

Reply:
Doesn't work in Firefox, except nightly build. But when it doesn't work, it's just a little more jumping